models/models.js

```diff
@@ -82,8 +82,36 @@ WHERE comments.article_id = $1;
     )
     .then((article) => {
       if (article.rows.length === 0) {
-        return Promise.reject({ status: 404, msg: "Not found" });
+        return Promise.reject({ status: 400, msg: "Not found" });
       }
       return article.rows;
     });
 };
+
+exports.addArticleCommentMod = (article_id, username, body) => {
+  return db
+    .query(
+      `
+    UPDATE articles 
```

Comment:
Like I said in previous comment, we should be making an `INSERT INTO` here because we are posting a comment!